.scripts/build_steps.sh

```diff
@@ -36,8 +36,16 @@ show_channel_urls: true
 solver: libmamba
 CONDARC
 
+# Workaround for errors related to "unsafe" directories:
+# https://github.blog/2022-04-12-git-security-vulnerability-announced/#cve-2022-24765 
+git config --global --add safe.directory "${FEEDSTOCK_ROOT}"
```

Comment:
Understand this on Linux in the Docker image. Though on macOS (where it is run on the user's system) should this be guarded?

Reply:
MacOS has its own script separate from this, it shouldn't be affected? Or do you mean when macOS is the host?